docs/languages/powershell.md

```diff
@@ -0,0 +1,209 @@
+# Editing PowerShell in Visual Studio Code
+
+PowerShell is a task-based command line shell and scripting language built on .NET .
```

Comment:
nitpick: there's an extra space after ".NET"

it should be:

"language built on .NET."